.github/workflows/phar-build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build PHAR
+
+on:
+    push:
+        branches: [ "main" ]
+    pull_request:
+        branches: [ "main" ]
+
+permissions:
+    contents: read
+
+jobs:
+    build-phar:
+        name: "Build PHAR"
+
+        runs-on: ${{ matrix.operating-system }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                dependencies:
+                    - "locked"
+                php-version:
+                    - "8.1"
+                operating-system:
+                    - "ubuntu-latest"
+
+        steps:
+            - name: "Checkout"
+              uses: "actions/checkout@v4"
+
+            - name: "Install PHP"
+              uses: "shivammathur/setup-php@v2"
+              with:
+                  coverage: none
+                  tools: composer:v2
+                  php-version: "${{ matrix.php-version }}"
+                  ini-values: memory_limit=-1
+
+            - name: "Get Composer Cache Directory"
+              id: composer-cache
+              run: |
+                  echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+            - name: "Cache Composer dependencies"
+              uses: "actions/cache@v3"
+              with:
+                  path: "${{ steps.composer-cache.outputs.dir }}"
+                  key: "php-${{ matrix.php-version }}-locked-composer-${{ hashFiles('**/composer.lock') }}"
+                  restore-keys: |
+                      php-${{ matrix.php-version }}-locked-composer-
+
+            - name: "Install locked dependencies"
+              run: "composer install --no-interaction --no-progress"
+
+            - name: "Build PHAR file"
+              run: "composer build:phar"
+
+            - name: "Test PHAR file"
+              run: "./spc.phar dev:extensions"
+
+            - uses: actions/upload-artifact@v3
+              with:
+                  path: spc.phar
```

Comment:
If it's just for introducing boxes, we don't seem to need the workflow of automatically packaging phar now.

Reply:
I would leave it, as it's a good base for https://github.com/crazywhalecc/static-php-cli/issues/179, but just remove for now (comment out) autostart on push, etc.